config/crd/bases/etos.eiffel-community.github.io_clusters.yaml

```diff
@@ -401,7 +401,7 @@ spec:
                     default: ""
                     type: string
                   host:
-                    default: eventrepository
+                    default: ""
```

Comment:
Whats the rationale behind this change?

Reply:
That default would never work so we added a better way to set the default down in the `eventrepository.go` file:
https://github.com/eiffel-community/etos/pull/410/files#diff-30f86aebbd880d7c00ec6a5e7a3e2869e21f9c1f4868f6a3618f4e6c195eca1dR98-R100